en-US/win10/samples/ZigBeeAdapterTutorial.md

```diff
@@ -0,0 +1,203 @@
+If needed, see instruction [here](http://ms-iot.github.io/content/en-US/win10/AppDeployment.htm) for remote debugging.
+ 
+ 
+### Known limitations of the current version of the ZigBee adapter
+- ZigBee adapter only discovers devices that are directly connected to the XBee module. 
+- ZigBee adapter only support some devices defined in ZigBee Light Link or Home Automation profiles. This means that it only implements the necessary ZCL clusters and ZDO commands to handle them. That said support for new devices and new ZCL clusters can easily be added to ZigBee adapter code.
+- ZigBee adapter doesn’t expose any method to commission ZigBee devices.
+
+## ZigBee adapter in detail 
+ZigBee adapter is written in C# and exposes ZigBee devices on AllJoyn through BridgeRT interface. ZigBee device is exposed in AllJoyn as follow:
+- Each ZigBee end point of a ZigBee device is exposed as an AllJoyn Service (bus attachment)
+- Each cluster of an end point is exposed as an AllJoyn Bus object
+- Each Attribute of a cluster is exposed as an AllJoyn Property
+- All ZigBee commands of all clusters of an end point are grouped under the main AllJoyn bus object of the AllJoyn Service and exposed as AllJoyn method
+
+![ZigBee2AllJoynMapping]({{site.baseurl}}/images/ZigBee/ZigBee2AllJoynMapping.png)
+
+### AllJoyn Explorer view of Philips Hue light bulb
+Philips Hue light bulb has 1 endpoint which has several clusters: Identify, Scene, Group, OnOff and LevelControl. ZigBee adapter only handles the OnOff and LevelControl clusters hence will only expose them to AllJoyn. Below is AllJoyn explorer view of what is exposed on AllJoyn. Path to “On” command is highlighted in green, path to “OnOff” status is highlighted in red.
+
+![AJXPhilipsHue1]({{site.baseurl}}/images/ZigBee/AJXPhilipsHue1.png)
+
+![AJXPhilipsHue2]({{site.baseurl}}/images/ZigBee/AJXPhilipsHue2.png)
+
+![AJXPhilipsHue3]({{site.baseurl}}/images/ZigBee/AJXPhilipsHue3.png)
+
+![AJXPhilipsHue4]({{site.baseurl}}/images/ZigBee/AJXPhilipsHue4.png)
+
+![AJXPhilipsHue5]({{site.baseurl}}/images/ZigBee/AJXPhilipsHue5.png)
+
+### Class overview
+
+![ClassMap]({{site.baseurl}}/images/ZigBee/ClassMap.png)
+
+The __Adapter__ class is the main class of ZigBee adapter. This class derives from __IAdapter__ (BridgeRT interface) and contains a collection of __ZigBeeDevice__ instances and an instance of the __XBeeModule__ class.
+
+The __XBeeModule__ class handles the interactions with XBee module, this class has method to build and parse XBee frames. See XBee ZigBee module documentation (http://www.digi.com) for more detail about its API and its frame layout. XBeeModule class uses the __SerialController__ class that handles communication over a serial port. 
+
+The __ZigBeeDevice__ class represent a ZigBee device. This class has no interface with BridgeRT since only EndPoints of a ZigBee device are expose to AllJoyn. __ZigBeeDevice__ class contains a collection of __ZigBeeEndPoint__ instances.
```

Comment:
Suggested grammar fixes:

The __ZigBeeDevice__ class represents a ZigBee device. This class has no interface with BridgeRT since only EndPoints of a ZigBee device are exposed to AllJoyn. The __ZigBeeDevice__ class contains a collection of __ZigBeeEndPoint__ instances.